.github/workflows/push.yml

```diff
@@ -0,0 +1,63 @@
+name: CI
+
+on:
+  push:
+    branches: [ master ]
+    tags:
+    - v*
+  pull_request:
+    branches: [ master ]
+  
+jobs:
+  build:
+    name: Build
+    runs-on: ubuntu-latest
+    steps:
+
+    - name: Check out code into the Go module directory
+      uses: actions/checkout@v2
+
+    - name: Set up Docker Buildx
+      id: buildx
+      uses: docker/setup-buildx-action@v1
+      with:
+        install: true
+        version: latest
+        
+    - name: Set up QEMU
+      id: qemu
+      uses: docker/setup-qemu-action@v1
+      with:
+        image: tonistiigi/binfmt:latest
+        platforms: all
+  
+    - name: Login to DockerHub
+      uses: docker/login-action@v1
+      with:
+        username: ${{ secrets.DOCKERHUB_USERNAME }}
+        password: ${{ secrets.DOCKERHUB_TOKEN }}
+ 
+    -
+      name: Login to GitHub Container Registry
+      uses: docker/login-action@v1 
+      with:
+        registry: ghcr.io
+        username: ${{ github.repository_owner }}
+        password: ${{ secrets.CR_PAT }}
+
+    -
+      name: Docker meta
+      id: docker_meta
+      uses: crazy-max/ghaction-docker-meta@v1
+      with:
+        images: ${{ github.repository_owner }}/instancemgr,ghcr.io/${{ github.repository_owner }}/instance-manager
+    -
+      name: Build and push
+      uses: docker/build-push-action@v2
+      with:
+        context: .
+        file: ./Dockerfile
+        platforms: linux/amd64,linux/arm/v7,linux/arm64
+        push: true
+        tags: ${{ steps.docker_meta.outputs.tags }}
```

Comment:
So which tags will this actually push?
Our current CI pushes an image with `master` tag on every merge to master branch, and on every release tag, it pushes the tag name as the image tag e.g. `0.9.0`

Reply:
From the GH Action docs:

Event | Ref | Commit SHA | Docker Tags
-- | -- | -- | --
pull_request | refs/pull/2/merge | a123b57 | pr-2
push | refs/heads/master | cf20257 | master
push | refs/heads/my/branch | a5df687 | my-branch
push tag | refs/tags/v1.2.3 | ad132f5 | v1.2.3, latest
push tag | refs/tags/v2.0.8-beta.67 | fc89efd | v2.0.8-beta.67, latest

Reply:
ok nice, so if I understand correctly, on master merge it would push `master` and on tag push it would push same tag name + `latest`.

Reply:
yep